modules/components/containers/RuleContainer.jsx

```diff
@@ -150,13 +161,13 @@ const createRuleContainer = (Rule) =>
               onDragStart={this.props.onDragStart}
               setLock={isInDraggingTempo ? this.dummyFn : this.setLock}
               removeSelf={isInDraggingTempo ? this.dummyFn : this.removeSelf}
-              setField={isInDraggingTempo ? this.dummyFn : this.setField}
+              setField={isInDraggingTempo ? this.dummyFn : this.setField} // TODO: needed in lhs?
               setOperator={isInDraggingTempo ? this.dummyFn : this.setOperator}
               setOperatorOption={isInDraggingTempo ? this.dummyFn : this.setOperatorOption}
               setValue={isInDraggingTempo ? this.dummyFn : this.setValue}
               setValueSrc={isInDraggingTempo ? this.dummyFn : this.setValueSrc}
-              selectedField={this.props.field || null}
-              parentField={this.props.parentField || null}
+              selectedField={selectedField || null} // TODO
+              parentField={this.props.parentField || null} // TODO
```

Comment:
TODO what exactly?

Reply:
Initially I thought I will have to adjust somehow all references to field and parent field. I haven't seen any issues related to `parentField` property yet, so I'm removing the comment